std/http/server_test.ts

```diff
@@ -590,4 +590,55 @@ test({
   }
 });
 
+test({
+  name: "[http] respond error handling",
+  async fn(): Promise<void> {
+    const connClosedPromise = deferred();
+    const serverRoutine = async (): Promise<void> => {
+      let reqCount = 0;
+      const server = serve(":8124");
+      const serverRid = server.listener["rid"];
+      let connRid = -1;
+      for await (const req of server) {
```

Comment:
Seems to be waiting for the next req (on my machine)

Reply:
Yeah that should be the reason. I wonder why the second `req.respond()` does not fail on windows...